content/scalers/postgresql.md

```diff
@@ -0,0 +1,91 @@
++++
+fragment = "content"
+weight = 100
+title = "PostgreSQL"
+background = "light"
++++
+
+Scale applications based on a PostgreSQL query
+
+**Availability:** v1.2+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `postgresql` trigger that scales based on a postgresql query
+
+The Postgresql scaler allows for two connection options:
+
+A user can offer a full connection string 
+(often in the form of an environment variable secret)
+
+- `connectionString` MySQL connection string that should point to environment variable with valid value
+
+Alternatively, a user can specify individual
+arguments (host, userName, password, etc.), and the scaler will form a connection string 
+internally.
+- `host:` Service URL to postgresql. Note that you should use a full svc URL as KEDA will need to contact postgresql from a different namespace
```

Comment:
Can you add a new line before listing please?

Currently the list is not rendered correctly:
![image](https://user-images.githubusercontent.com/4345663/74008681-c3095900-4981-11ea-9154-64dcd7031196.png)

https://deploy-preview-71--vigorous-franklin-8edd65.netlify.com/scalers/postgresql/